src/jv.h

```diff
@@ -70,6 +76,8 @@ jv jv_true(void);
 jv jv_false(void);
 jv jv_bool(int);
 
+static inline jv jv_is_borrowed(jv a){if(a.borrowed) return jv_true(); return jv_false();}
```

Comment:
Does anything need this function?  I think probably not.

Reply:
while I can't think of much of a usecase I implemented it just in case

Reply:
Also it is to have for the sake of consistency as jv_get_refcount() also exists

Reply:
This function is actually needed a lot(at least within jv) Outside everything should be able to be done with jv_return() and jv_unborrow() , but I think this ought to stay there